### ISSUE TITLE
How to use lampe in pure command line?

### ISSUE THREAD

**Issue by NerosTie** [poinck/lampe#34]
I want to add keyboard shortcuts for `lampe` in KDE or add aliases for my terminal. Just to switch on/off. But I don't know how to use `lampe` in pure command line because the only way to use it is to launch the bash-script interface.

Is there a possibility?

**Comment by poinck** [comment]
Currently it just supports interactive-mode. In early days I started with simple batch-mode. I have to look wether it can be easly achieved to support both:

So, something like this (brainstorming; option `-s` for "set" and quit):
```.sh
lampe -s 2 y # light 2 on
lampe -s 1 n # light 1 off
lampe -s y # switch all lights on
```
would help you?

**Comment by NerosTie** [comment]
Yes! It would be great! :+1:

**Comment by poinck** [comment]
I will start something in a seperate branch. I'll let you know as soon there is something to try out.

**Comment by poinck** [comment]
You can now test the new oneshot-mode from branch ["issue34-oneshot_mode"](https://github.com/poinck/lampe/tree/issue34-oneshot_mode).

All options should work, but I would not recommend starting sequences in oneshot-mode (needs further testing); I have tried following to start and stop blinking, which worked:

```
lampe -s 5 b
lampe -s Q # to end all sequences or
lampe -s 5 b # to just stop blinking of light 5
```

Have fun! (:

**Comment by poinck** [comment]
pull-request #35 waits for testing.

**Comment by poinck** [comment]
tested and merged to master, will be released with version 1.1.6
